src/App.js

```diff
@@ -1,26 +1,9 @@
-import logo from './logo.svg';
 import './App.css';
+import Calculator from './components/calculator';
 
 function App() {
   return (
-    <div className="App">
```

Comment:
- [ ] The App.js component should be class-based as per the project's requirement. In this milestone, the objective is to practice the class-based React component. :pray:

Reply:
This issue is well addressed. Great work! :clap: